training/trainer.py

```diff
@@ -192,6 +192,7 @@ def train(
     deepspeed: str,
     gradient_checkpointing: bool,
     local_rank: str,
+    fp16: bool,
```

Comment:
Let's assume either fp16 or bf16 are true, so bf16 being false implies fp16 is true

Reply:
Makes sense, I'll put this change in.